product_docs/docs/eprs/7.0/10_appendix/04_miscellaneous_xdb_processing_topics/01_publications_and_subscriptions_server_conf_options/07_assign_ip_adress_for_rmi.mdx

```diff
@@ -1,15 +1,15 @@
 ---
-title: "Assigning an IP Address for Remote Method Invocation"
+title: "Assigning an IP address for remote method invocation"
 ---
 
 <div id="assign_ip_adress_for_rmi" class="registered_link"></div>
 
 !!! Note
-    The option described in this section applies to the publication server and the subscription server.
+    This option applies to the publication server and the subscription server.
 
 **For Linux only:**
 
-An alternative method to modifying the `/etc/hosts` file so that the host name is associated with a non-loopback IP address as discussed in Section [Network IP Addressese](../../../05_smr_operation/01_prerequisites/06_verifying_host_accessibility/#network_ip) is to specify the network IP address using the java.rmi.server.hostname option.
+An alternative method to modifying the `/etc/hosts` file so that the host name is associated with a non-loopback IP address, as discussed in [Network IP addressese](../../../05_smr_operation/01_prerequisites/06_verifying_host_accessibility/#network_ip), is to specify the network IP address using the `java.rmi.server.hostname` option.
```

Comment:
```suggestion
An alternative method to modifying the `/etc/hosts` file so that the host name is associated with a non-loopback IP address, as discussed in [Network IP addresses](../../../05_smr_operation/01_prerequisites/06_verifying_host_accessibility/#network_ip), is to specify the network IP address using the `java.rmi.server.hostname` option.
```